### PR TITLE
Fix ssdloss num and batch norm format and conv2d

### LINKED ISSUE #19551: BatchNorm中，若batchSIze为1的输入，会怎么处理
训练时，BatchNorm中，若batchSize为1的输入，会怎么处理。按照公式的话均值是他自身，减完自身不就变成全0了吗。若是保持不变的话，那inference时减均值后就和训练时的数据完全不同了

### PR DESCRIPTION
**Pull request by ceci3**
ssd_loss 5.1 --> 5.2
fix https://github.com/PaddlePaddle/Paddle/issues/19551 fix batch  norm format
fix conv doc

### CODE REVIEW

**Review by heavengate — DISMISSED** [review]
LG API

**Review by wanghaoshuang — DISMISSED** [review]
LG on API

**Review by heavengate — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]
